pwa/src/templates/templateParts/schemasForm/SchemaFormTemplate.tsx

```diff
@@ -40,7 +40,19 @@ export const SchemaFormTemplate: React.FC<SchemaFormTemplateProps> = ({ schema }
   React.useEffect(() => {
     if (!schema) return;
 
-    ["name", "description", "reference"].forEach((field) => setValue(field, schema[field]));
+    [
+      "name",
+      "description",
+      "reference",
+      "version",
+      "maxDepth",
+      "extend",
+      "inherited",
+      "exclude",
+      "persist",
+      "createAuditTrails",
+      "softDelete",
```

Comment:
When sending exclude and softDelete to the backend, they don't come back when we ask for the object.
I think the backend does not use these properties yet.